### ISSUE TITLE
The globalKeyboardListener should be removed from PlayPauseButton

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/scenery-phet#667]
It shouldn't be in this class because there is no constraint that there is only one in a sim. Anything using more than one PlayPauseButton will have all button isPlayingProperties toggle with "Alt + k".  TimeControlNode might work, but what if a sim uses more than one of those as well? Maybe we need to take that listener out and add it on a sim by sim basis.

@zepumph can we brainstorm a pattern for global keyboard listeners like this soon?

**Comment by zepumph** [comment]
How about we wrap that listener in an option, auto on, so that you can turn if off if we encounter a case for the second play pause button?

**Comment by jessegreenberg** [comment]
OK thanks, that sounds good for now. Can you please review, particularly the name of the option?

**Comment by zepumph** [comment]
Very nice!

**Comment by jessegreenberg** [comment]
OK thanks!
